buildlib/pr/cuda/cuda.yml

```diff
@@ -85,6 +85,22 @@ jobs:
           CONTAINER: ubuntu20_cuda_11_5
         ubuntu20_cuda_11_6:
           CONTAINER: ubuntu20_cuda_11_6
+        centos7_cuda_12_0:
```

Comment:
CentOS 8 images with CUDA 12 are not available:
https://hub.docker.com/r/nvidia/cuda/tags?page=1&name=12.0.0-devel-centos
Probably because CentOS 8 is EOL.

Reply:
For CentOS/RHEL we should use RedHat UBI images https://hub.docker.com/r/redhat/ubi8

Reply:
I think we can add ubiX as a separate PR

Reply:
ok

Reply:
Please, specify what UBI and CUDA versions are required.
Available versions:
UBI: [7-9]
CUDA: [10.0-12.1]